clap_derive/src/derives/args.rs

```diff
@@ -170,6 +330,9 @@ pub(crate) fn gen_augment(
     fields: &[(&Field, Item)],
     app_var: &Ident,
     parent_item: &Item,
+    // when generating mutably exclusive arguments,
+    // ids of arguments that should conflict
+    conflicts: &[Name],
```

Comment:
Not super clean to sqeeze this in here esp, since this is also used by the `subcommand` deriver, but i tried to avoid rewriting the entire method for this.